parm/archive/master_enkf.yaml.j2

```diff
@@ -62,22 +62,22 @@ datasets:
 # This must be done in a namespace to overcome jinja scoping
 #  Variables set inside of a for loop are lost at the end of the loop
 #  unless they are part of a namespace
-{% set com_ns = namespace(COM_ATMOS_ANALYSIS_MEM = COM_ATMOS_ANALYSIS_TMPL,
-                          COM_ATMOS_HISTORY_MEM = COM_ATMOS_HISTORY_TMPL,
-                          COM_ATMOS_RESTART_MEM = COM_ATMOS_RESTART_TMPL) %}
+{% set com_ns = namespace(COMIN_ATMOS_ANALYSIS_MEM = COMIN_ATMOS_ANALYSIS_TMPL,
+                          COMIN_ATMOS_HISTORY_MEM = COMIN_ATMOS_HISTORY_TMPL,
+                          COMIN_ATMOS_RESTART_MEM = COMIN_ATMOS_RESTART_TMPL) %}
```

Comment:
These should be `COM_ATMOS_<>_TMPL`. 
This was the cause of failure reported in https://github.com/NOAA-EMC/global-workflow/pull/2667#issuecomment-2176655888

Reply:
Fixed.

Reply:
This PR was merged.  The fix needs a hotfix.